SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditor.cpp

```diff
@@ -305,17 +305,11 @@ bool FSpatialGDKEditor::LoadPotentialAssets(TArray<TStrongObjectPtr<UObject>>& O
 		}
 		Progress.EnterProgressFrame(1, FText::Format(LOCTEXT("LoadingSingleAsset_Text", "Loading {0}"), FText::FromName(Data.AssetName)));
 
-		const FString* GeneratedClassPathPtr = nullptr;
-
 		FAssetDataTagMapSharedView::FFindTagResult GeneratedClassFindTagResult = Data.TagsAndValues.FindTag("GeneratedClass");
 		if (GeneratedClassFindTagResult.IsSet())
 		{
-			GeneratedClassPathPtr = &GeneratedClassFindTagResult.GetValue();
-		}
-
-		if (GeneratedClassPathPtr != nullptr)
```

Comment:
Do we no longer care about this failure case?

Reply:
I think this failure case was fully covered by `if (GeneratedClassFindTagResult.IsSet())`, since GeneratedClassPathPtr was being set to the address of a value we got for sure if the tag result was set.